### ISSUE TITLE
image normalization

### ISSUE THREAD

**Issue by sean7218** [nyoki-mtl/keras-facenet#12]
i was able to create the embedding using the converted CoreML keras model but the input image was not normalized thus the output was incorrect. 

I used the tensorflow method `tf.image.per_image_standardization(image)` to train the data thus would like to use the similar method. Do I need to implement it myself in Swift? or Should I add the layer in the keras model ?
